Venmo/src/main/java/com/braintreepayments/api/VenmoPaymentAuthRequest.kt

```diff
@@ -0,0 +1,17 @@
+package com.braintreepayments.api
+
+/**
+ * A request used to launch the Venmo app for continuation of the Venmo payment flow.
+ */
+sealed class VenmoPaymentAuthRequest {
```

Comment:
nit: There's [some interesting interop](https://stackoverflow.com/a/53771710) cases between Kotlin and Java with sealed classes.